src/NovelAudioService.h

```diff
@@ -28,6 +29,7 @@ typedef std::map<std::string, int> ChannelMap;
   SoundBank _sounds;
   MusicBank _music;
   ChannelMap _channelMap;
+  NovelLoggingService _console;
```

Comment:
Maybe change the name of this to `_logger` instead?

Reply:
Fixed!

Reply:
Err, for each class holding onto the LoggingService, its listed as `_console` -> should the others change as well?